builtins.isph

```diff
@@ -0,0 +1,685 @@
+// -*- mode: c++ -*-
+/*
+  Copyright (c) 2024, Intel Corporation
+
+  SPDX-License-Identifier: BSD-3-Clause
+*/
+
+/** @file builtins.isph
+
+    @brief This file contains the declarations of the built-in functions that
+    are defined in builtins*.ll files but used in the ISPC code of standard
+    library. These functions should not be used outside of the standard library
+    code.
+*/
+
+extern "C" varying int32 __padds_vi32(varying int32, varying int32);
```

Comment:
Please sort them alphabetically.

Reply:
I have sorted them alphabetically inside groups of functions and reordered these groups.